app/controllers/taggings_controller.rb

```diff
@@ -1,14 +1,12 @@
 class TaggingsController < ApplicationController
   before_action :authenticate_user!
-  before_action :set_movie
+  before_action :set_movie, only: [:create, :destroy]
```

Comment:
I'm making this explicit because I think that it makes it easier to understand scope and reduces risk of this action happening for other actions added later.